src/plugins/plot/axis/YAxis.vue

```diff
@@ -107,23 +102,71 @@ export default {
         }
     },
     mounted() {
-        this.yAxis = this.getYAxisFromConfig();
+        eventHelpers.extend(this);
+        this.initAxisAndSeriesConfig();
         this.loaded = true;
         this.setUpYAxisOptions();
     },
     methods: {
-        getYAxisFromConfig() {
+        initAxisAndSeriesConfig() {
             const configId = this.openmct.objects.makeKeyString(this.domainObject.identifier);
             let config = configStore.get(configId);
             if (config) {
-                return config.yAxis;
+                if (!this.id || this.id === config.yAxis.id) {
+                    this.yAxis = config.yAxis;
+                } else {
+                    this.yAxis = config.additionalYAxes.find(yAxis => yAxis.id === this.id);
+                }
+
+                this.config = config;
+                this.listenTo(this.config.series, 'add', this.addSeries, this);
+                this.listenTo(this.config.series, 'remove', this.removeSeries, this);
+                this.listenTo(this.config.series, 'reorder', this.addOrRemoveSeries, this);
+
+                this.config.series.models.forEach(this.addSeries, this);
             }
         },
+        addOrRemoveSeries(series) {
+            const yAxisId = this.series.get('yAxisId');
+            if (yAxisId === this.id) {
+                this.addSeries(series);
+            } else {
+                this.removeSeries(series);
+            }
+        },
+        addSeries(series, index) {
+            const yAxisId = series.get('yAxisId');
+            const seriesIndex = this.seriesModels.findIndex(model => this.openmct.objects.areIdsEqual(model.get('identifier'), series.get('identifier')));
+
+            if (yAxisId === this.id && seriesIndex < 0) {
+                this.seriesModels.push(series);
+                this.checkRangeValueAndSingleSeries();
+                this.setUpYAxisOptions();
+            }
+        },
+        removeSeries(plotSeries) {
+            const seriesIndex = this.seriesModels.findIndex(model => this.openmct.objects.areIdsEqual(model.get('identifier'), plotSeries.get('identifier')));
+            if (seriesIndex > -1) {
+                this.seriesModels.splice(seriesIndex, 1);
+                this.checkRangeValueAndSingleSeries();
+                this.setUpYAxisOptions();
+            }
+        },
+        checkRangeValueAndSingleSeries() {
+            this.hasSameRangeValue = this.seriesModels.every((model) => {
```

Comment:
Should these be computed values instead?

Reply:
No, this is specifically not computed. The seriesModels property will become Vueified if we do that and we don't want it to. This works fine.